### ISSUE TITLE
Error reading ANDOR tags in tiffread31_readtags

### ISSUE THREAD

**Issue by bpisaacoff** [DylanMuir/TIFFStack#24]
Hi Dylan, thank you so much for TIFFStack this has sped up my work so much! 

I've just updated to Matlab 2016b and re-downloaded this code (within the week). All of a sudden I'm getting an error, which after much chasing, is coming from the Andor tags section in the tiffread31_readtags function. Sometimes different tags, with the attached file it's with the exposureTime tag. Specifically I'm getting the following error (after having commented out the try and catch in TIFFStack.m):
```
A dot name structure assignment is illegal when the structure is empty.  Use a
subscript on the structure.

Error in tiffread31_readtags (line 154)
            sTags(img_indx).ANDOR.exposureTime   = entry_val;

Error in TIFFStack (line 241)
            [bIsImageJBigStack, bIsImageJHyperStack, vnStackDims,
            vnInterleavedIJFrameDims] =
            IsImageJBigStack(tiffread31_readtags(oStack.TIF_tr31, oStack.HEADER,
            1), numel(oStack.HEADER));
```
[forDylanMuir.zip](https://github.com/DylanMuir/TIFFStack/files/714854/forDylanMuir.zip)

We were able to work around this by just commenting out the whole Andor tags section in tiffread31_readtags

Any idea what is going on and why this happened all of a sudden?

Thanks a lot,

**Comment by DylanMuir** [comment]
Hi @bpisaacoff, I have pushed a commit that fixes this issue, and allows you to read in the ANDOR tags.

Thanks for flagging the issue!
Dylan.

**Comment by bpisaacoff** [comment]
Awesome thanks so much for the quick fix. I'll be sure to let you know if we
have further issues.
------------------------Ben Isaacoff






On Wed, Jan 18, 2017 3:39 PM, Dylan Muir notifications@github.com
wrote:
Closed #24.
